embedded/logger/simple_test.go

```diff
@@ -63,13 +62,24 @@ func TestSimpleLogger(t *testing.T) {
 func TestLogLevelFromEnvironment(t *testing.T) {
```

Comment:
this test seems to be failing

Reply:
Ahh...that's because `make test` sets `LOG_LEVEL=error` :sweat_smile: And prior to this PR we have been doing `defer os.Unsetenv("LOG_LEVEL")` which unsets the `LOG_LEVEL=error` when the tests finished.

https://github.com/codenotary/immudb/blob/0ede732721dec111736051fbb0cc1e996e41b8a5/Makefile#L135-L138

![image](https://github.com/codenotary/immudb/assets/20135478/a3984a43-7d0f-456b-96df-4fa86bb47a3c)

Reply:
Fixed in 1da96b07c2b49786f15477eabf1c9d87dc450bb5